### DISCUSSION TITLE
basic_handshaking_test.py

### DISCUSSION THREAD

**Discussion by iidyc** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#8] [Q&A] [answered]
basic_handshaking_test.py
第10行: peer2的chunkfile好像应该为data2.fragment?
第29行: assert XXX == 0改为XXX > 0也许更合理，因为有些人的实现可能不止发一次WHOHAS

**Comment by TANSixu** [comment, marked as answer]
非常感谢，已修改！
见PR #9 
再次感谢！
